packages/nitro-protocol/gas-benchmarks/gas.ts

```diff
@@ -47,7 +49,16 @@ export const gasRequiredTo: GasRequiredTo = {
       // ^^^^^
       // In principle this only needs to be done once per account
       // (the cost may be amortized over several deposits into this AssetHolder)
-      deposit: 55754,
+      deposit: 55766,
     },
   },
+  ETHexit: {
+    // e completely liquidate the channel (paying out both parties)
+    vanillaNitro: 146793,
+  },
+  ERC20exit: {
+    // We completely liquidate the channel (paying out both parties)
+    // This is cheaper than the ETHexit because storage slots are zero-ed out.
+    vanillaNitro: 124148,
+  },
```

Comment:
Understanding check:

`ETHexit` consumes more gas than `ERC20exit`, despite ETH transfers being privileged by the EVM, because of differences in the setup:

- `ETHAssetHolder.deposit` writes a new storage slot in the assetholder (`holdings[destination]` is being upped from 0 to 10), while
- ... (20 minutes later) actually I don't seem to understand this. Comment?

Reply:
I think you are broadly on the correct track. 

ETH deposits and payouts require no writes to storage other than the `AssetHolder.holdings` mapping.

ERC20 deposits and payouts require writes to storage in the Token contract itself. The first deposit is very expensive (20K), but if we pair it with the refund from zeroing the slot when paying out (-15K) it's only really 5K more expensive. 

What we are seeing here is the ERC20 exit benefitting from such a refund. If we think about that in isolation, it seems cheaper than an ETH exit. But when we look above, we see that the ERC20 deposit carried a gas premium that outweighs that refund. 

EIP-3529 is going to greatly reduce the refund. 

However, you have made me realise that this benchmark is probably not a realistic one. We only get to zero the balance of the AssetHolder (in the Token contract) when there are no other funded channels in the universe. That is very unlikely. We should alter the benchmark to keep some ERC20 tokens in the AssetHolder (for another channel, say).

Reply:
I made the change in cb05d14; for the ETH case too just for completeness (it makes no difference, as expected)

Reply:
It results in exactly a 15K increase :-)